Add Website Screenshots

deadcoder0904 opened the pull request:
<!-- Please fill in the **bold** fields -->

**https://github.com/deadcoder0904/awesome-website-screenshots**

**Awesome Website Screenshots is a curation of some projects, websites or mockups that allow to create different screenshots in different resolutions with different frames.**


# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

sindresorhus commented:
Good start, but I think the list needs more work. I'd like to see some more resources like blog posts, tutorials, YouTube videos, etc.

Feedback:

- `It is a Command Line Interface Tool for the above project Pageres.` => `Command-line tool for the above project Pageres.`
- `and take screenshots of each one.` => `and takes screenshots of each one.`
- `url` => `URL`
- `Provides an Interface` => `Provides an interface` (You're uppercasing a lot of words that shouldn't be. Go through the list and fix all those instances)
- `photoshop` => `Photoshop`

sindresorhus commented on readme.md:
```diff
@@ -468,8 +468,8 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
-- [Web Design](https://github.com/nicolesaidy/awesome-web-design) - Resources for digital designers. 
-
+- [Web Design](https://github.com/nicolesaidy/awesome-web-design) - Resources for digital designers.
+- [Website Screenshots](https://github.com/deadcoder0904/awesome-website-screenshots) - Take Website Screenshots on different devices with different resolutions in different frames.
```
Needs a shorter description.

deadcoder0904 commented:
Yep I'll make changes :smile:

sindresorhus commented:
ping

deadcoder0904 commented:
Ohh sorry for the delay :smile: I'll make the necessary changes in a week currently very busy on another project which is due this week so I will need some time.. I'll make the changes this week for sure :+1: Thank You for the patience

deadcoder0904 commented:
I've added blog posts (which are very few) as there aren't many blogs out there on this particular topic or they don't provide unique content. Also I've made all the necessary changes, fixed all my Uppercases :joy: & provided a shorter description..Thank You :smile:

sindresorhus commented:
I still feel the list could use more work. There are still many typos in the description and the formatting could be improved. I think you should split the `Websites` section into subsections for device mockups and for just the device sizes. You could also list some useful browser extensions for capturing screenshots.

I'm gonna close this for now. Try again in a few months if the list has improved considerably :)

---

"5 Brilliant Ways to Add Device Frames to Your App Screenshots" - This is click-baity, low-quality and outdated. When I said articles, I meant awesome ones. This definitely isn't.

deadcoder0904 commented:
As far as I've checked & I've said this before "There aren't enough resources on this topic" & as far as Awesome Ones are concerned this is what I found in articles & that article u mentioned is definitely low-quality :stuck_out_tongue_closed_eyes: as there isn't anything to add further on this topic...I'll send a PR if I think this list is improved in a years time....But as I said this is a topic which has very few links or isn't indexed by Google Crawlers.. Cyaa :wink:
